gradle/artifact-settings.gradle

```diff
@@ -10,5 +10,5 @@ ext {
     mapboxArtifactScmUrl = 'scm:git@github.com:mapbox/mapbox-navigation-android.git'
     mapboxArtifactLicenseName = 'Mapbox Terms of Service'
     mapboxArtifactLicenseUrl = 'https://www.mapbox.com/legal/tos/'
-    versionName = System.getenv('POM_VERSION_NAME') ?: project.property('POM_SNAPSHOT_VERSION_NAME')
+    versionName = System.getenv('POM_VERSION_NAME') ?: project.hasProperty('VERSION_NAME') ? project.property('VERSION_NAME') : project.property('POM_SNAPSHOT_VERSION_NAME')
```

Comment:
Could you instead refactor `config.yml` to change the setup to `VERSION_NAME` and drop the duplication?

Reply:
sure! wanted to make the change as small as possible but happy to rework fully.